NEWS.md

```diff
@@ -1,86 +1,118 @@
+## 2.0.0 PENDING
+
+- Upgrade to Spring and OpenAPI
+- Add sample data
+- Add more extensible calendar and exception storage format
+- Performance improvements
+- Add translation capabilities
+- Provide more informative API error messages
+- Rename permissions to new schema
+- Replace API routes with new schema
```

Comment:
Why was hyphen chosen over asterix when that is the general FOLIO convention?

Reply:
There wasn't any reason, I've restored the original formatting.